frontend/src/types/group.ts

```diff
@@ -0,0 +1,20 @@
+export enum GroupType {
+  DAHub = 'DA_HUB',
+  ReceivingGroup = 'RECEIVING_GROUP',
+  SendingGroup = 'SENDING_GROUP',
+}
+
+export type GroupLocation = {
+  countryCode?: string
+  townCity: string
+  openLocationCode?: string
+}
+
+export type Group = {
+  id: number
+  name: string
+  groupType: GroupType
+  primaryLocation: GroupLocation
+  primaryContactName: string
+  website?: string
+}
```

Comment:
Suggestion (blocking): I think we should be able to import these types directly from the common schema. From the video it sounds like a front end build or dev tool requires that all files for it be available in the `frontend` subdir. That's doable by editing the `codegen.yaml` in the root dir. Try changing like 9 from `./api-types.ts:` to `frontend/api-types.ts:`or `frontend/src/api-types.ts:` (whatever you prefer).

I'd be really nice if we could use a common source of truth between the client and server for defining the shape of data.

Reply:
I agree about the common source of truth, I'll give it a shot 👍

Reply:
@bion I made the changes you suggested in https://github.com/distributeaid/shipment-tracker/pull/44/commits/9fffcb9b13a9141f61e9fe3ce2b0050dd9496e74.